docs/src/examples/supplemental_material/Convex.jl_intro_ISMP2015.jl

```diff
@@ -62,24 +62,25 @@ using Interact, Plots
     solve!(problem, SCS.Optimizer; silent_solver = true)
     histogram(evaluate(x), xlims = (0, 3.5), label = "x")
 end
+nothing # hide
```

Comment:
this doesn't produce anything that shows up in the rendered docs (and maybe never did, I think it was a notebook before), and also causes Documenter to warn about a big block that it can't render as svg. So I add `nothing` so it doesn't try to render anything.